src/ClientConfig.js

```diff
@@ -5,18 +5,18 @@ var igeClientConfig = {
 		'/lib/msgpack.min.js',
 
 		'/gameClasses/Player.js',
-		'/gameClasses/Unit.js',
+		// '/gameClasses/Unit.js',
```

Comment:
how come game classes are not being loaded anymore?

Reply:
loaded in physics because they depend on which version of IgeEntityPhysics.js

Reply:
seems like the same game classes get loaded regardless of the physics engine used:

https://github.com/moddio/taro/blob/8acdd33560c020e94b0acc4c9fafe9b19735a8d1/server/server.js#L452
https://github.com/moddio/taro/blob/8acdd33560c020e94b0acc4c9fafe9b19735a8d1/engine/PhysicsConfig.js#L62
https://github.com/moddio/taro/blob/8acdd33560c020e94b0acc4c9fafe9b19735a8d1/engine/PhysicsConfig.js#L47

and it doesn't seem like it should be physics loader's responsibility to load game classes anyways.

Reply:
yeah this may be left over from an earlier iteration where I had yet to load the game json before doing everything else in client.

Reply:
@pavle-goloskokovic there are things that can be done to make client loading better, but some are not necessary for current deployment.

Reply:
Just tried to make this change, and we get an error
```
var Unit = IgeEntityPhysics.extend({

ReferenceError: IgeEntityPhysics is not defined
```